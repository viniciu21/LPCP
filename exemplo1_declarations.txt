declaration:
    a: int; 
    b: int;
    c: int;
    d: int;
end_declaration