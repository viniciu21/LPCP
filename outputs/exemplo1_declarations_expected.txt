Declaração de variável: Id "a" (2,5)(True,[(Id "a" (2,5),IntValue 0 (2,8))],[],[],[])
Declaração de variável: Id "b" (3,5)(True,[(Id "a" (2,5),IntValue 0 (2,8)),(Id "b" (3,5),IntValue 0 (3,8))],[],[],[])
Declaração de variável: Id "c" (4,5)(True,[(Id "a" (2,5),IntValue 0 (2,8)),(Id "b" (3,5),IntValue 0 (3,8)),(Id "c" (4,5),IntValue 0 (4,8))],[],[],[])
Declaração de variável: Id "d" (5,5)(True,[(Id "a" (2,5),IntValue 0 (2,8)),(Id "b" (3,5),IntValue 0 (3,8)),(Id "c" (4,5),IntValue 0 (4,8)),(Id "d" (5,5),IntValue 0 (5,8))],[],[],[])
"Error message" (line 1, column 1):
unexpected end of input